src/Workspaces/Core/Portable/NamingStyles/NamingStyle.cs

```diff
@@ -314,6 +315,14 @@ private string FinishFixingName(string name)
                 words = name.Split(new[] { WordSeparator }, StringSplitOptions.RemoveEmptyEntries);
             }
 
+            if (CapitalizationScheme == Capitalization.AllLower && !string.IsNullOrEmpty(WordSeparator))
```

Comment:
Remove the second check and put inside the block above?

Reply:
Done